plugins/ring/src/main.ts

```diff
@@ -151,6 +151,16 @@ class RingPlugin extends ScryptedDeviceBase implements DeviceProvider, DeviceDis
         },
         refreshToken: {
             hide: true,
+        },
+        locationIds: {
+            title: 'Location ID',
+            description: 'Optional: If supplied will on show this locationID.',
```

Comment:
what does location id do?

Reply:
So when a locationId is supplied, it will then only show devices for that locationId

Before:<img width="854" alt="Screen Shot 2022-01-19 at 10 42 09 PM" src="https://user-images.githubusercontent.com/9875439/150274847-ff76a01e-f48a-4d48-9d4a-65b7bc9222b8.png">

After: <img width="862" alt="Screen Shot 2022-01-19 at 10 42 32 PM" src="https://user-images.githubusercontent.com/9875439/150274887-760e8799-0acf-4b24-9222-11696db6353c.png">

Reply:
What is the reason to do this? Generally it's fine to have everything imported, and then filter which services (Homekit, google home) get which synced devices.

Reply:
If someone accidentally checks a camera, then it will get added again. Where this they can manually set the locationIds of the cameras.

Like for my use case. I will never use the cameras I removed using this. Since they are from a shared account. Those cameras wouldn't be part of my house, but someone else's.

Reply:
I think it is fine to have this code, for debugging. I am going. To hide the setting. You can still access it by changing thread storage value directly.
I don’t think this is Useful to normal users who may fill this field out and then not see any devices.